docs/content/en/docs/pipeline-stages/cleanup.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Cleanup"
+linkTitle: "Cleanup"
+weight: 60
+featureId: cleanup
+---
+
+Skaffold works with [image builders]({{<relref "/docs/pipeline-stages/builders">}}) and [deployers]({{<relref "/docs/pipeline-stages/deployers">}}) 
+that both have side effects on the environment: 
+
+- resources are created in one or more namespaces in a Kubernetes cluster 
+- images might be created on the local Docker daemon
+- images might be pushed to registries
+- application might have arbitrary side effects 
+    
+Skaffold offers functionality to cleanup some of these side effects:
+
+- Kubernetes resource cleanup - `skaffold delete`, and Ctrl+C for `skaffold dev` and `skaffold debug`   
+- Image pruning - for local Docker daemon images only, on Ctrl+C for `skaffold dev` and `skaffold debug` 
+
+For pushed images in registries and application side effects the user has to take care of cleanup. 
```

Comment:
```suggestion
{{< alert title="Note" >}}
For safety reasons, Skaffold will **never** delete images pushed to any remote registry. Periodically, it's recommended to clean these up manually to avoid incurring excess storage costs.
{{</alert>}}
```

Reply:
why remove? Isn't it true? Transparency about this was my whole point.